etc/kayobe/kolla/globals.yml

```diff
@@ -50,46 +50,48 @@ caso_projects:
 # Container image tags.
 # These should be allowed to change independently, as services are updated.
 bifrost_tag: 7.0.1.10
-caso_tag: 7.0.1.8
-cron_tag: 7.0.1.6
-ceph_tag: 7.0.1.6
-cinder_tag: 7.0.1.6
-chrony_tag: 7.0.1.6
-elasticsearch_tag: 7.0.1.6
-fluentd_tag: 7.0.1.9
-glance_tag: 7.0.1.6
-grafana_tag: 7.0.1.6
-haproxy_tag: 7.0.1.6
-heat_tag: 7.0.1.6
-horizon_tag: 7.0.1.8
-influxdb_tag: 7.0.1.6
+caso_tag: 7.0.1.10
+cron_tag: 7.0.1.10
+ceph_tag: 7.0.1.10
+cinder_tag: 7.0.1.10
+chrony_tag: 7.0.1.10
+designate_tag: 7.0.1.10
+elasticsearch_tag: 7.0.1.10
+fluentd_tag: 7.0.1.10
+glance_tag: 7.0.1.10
+grafana_tag: 7.0.1.10
+haproxy_tag: 7.0.1.10
+heat_tag: 7.0.1.10
+horizon_tag: 7.0.1.10
+influxdb_tag: 7.0.1.10
 ironic_tag: 7.0.1.10
-iscsid_tag: 7.0.1.6
-kafka_tag: 7.0.1.6
-keepalived_tag: 7.0.1.6
-keystone_tag: 7.0.1.9
-kibana_tag: 7.0.1.6
-kolla_toolbox_tag: 7.0.1.6
-logstash_tag: 7.0.1.6
-magnum_tag: 7.0.1.6
-manila_tag: 7.0.1.6
-mariadb_tag: 7.0.1.6
-memcached_tag: 7.0.1.6
-monasca_tag: 7.0.1.6
-monasca_notification_tag: 7.0.1.6
-monasca_persister_tag: 7.0.1.6
+iscsid_tag: 7.0.1.10
+kafka_tag: 7.0.1.10
+keepalived_tag: 7.0.1.10
+keystone_tag: 7.0.1.10
+kibana_tag: 7.0.1.10
+kolla_toolbox_tag: 7.0.1.10
+logstash_tag: 7.0.1.10
+magnum_tag: 7.0.1.10
+manila_tag: 7.0.1.10
+mariadb_tag: 7.0.1.10
+memcached_tag: 7.0.1.10
+monasca_tag: 7.0.1.10
+monasca_notification_tag: 7.0.1.10
+monasca_persister_tag: 7.0.1.10
 neutron_tag: 7.0.1.10
 nova_tag: 7.0.1.10
 openvswitch_tag: 7.0.1.10
-prometheus_tag: 7.0.1.6
-prometheus_server_tag: 7.0.1.6
-prometheus_mtail_tag: 7.0.1.6
-prometheus_elasticsearch_exporter_tag: 7.0.1.6
+prometheus_tag: 7.0.1.10
+prometheus_server_tag: 7.0.1.10
+prometheus_mtail_tag: 7.0.1.10
+prometheus_elasticsearch_exporter_tag: 7.0.1.10
 prometheus_libvirt_exporter_tag: 7.0.1.6
```

Comment:
We should probably make sure this is building too, but thats a follow on thing before we do the stein upgrade

Reply:
Yeah, I'll make sure those images are built now.